#! /bin/sh /usr/share/dpatch/dpatch-run
## 08_strcasestr.dpatch by  <jdong@ubuntu.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Remove faulty strcasestr declaration to fix FTBFS
## DP: Thanks to Urs Fleisch for patch (LP: #444555)

@DPATCH@
diff -urNad mpeg4ip-1.6dfsg~/include/mpeg4ip.h mpeg4ip-1.6dfsg/include/mpeg4ip.h
--- mpeg4ip-1.6dfsg~/include/mpeg4ip.h	2006-08-03 14:21:43.000000000 -0400
+++ mpeg4ip-1.6dfsg/include/mpeg4ip.h	2009-10-06 11:22:36.366737222 -0400
@@ -123,7 +123,6 @@
 #ifdef __cplusplus
 extern "C" {
 #endif
-char *strcasestr(const char *haystack, const char *needle);
 #ifdef __cplusplus
 }
 #endif
